docs/tools/offline/cli-test-debug.md

```diff
@@ -79,6 +86,32 @@ Test Case:
     Path: .\TESTS\functional\test3
 ```
 
+And in case of icetea:
```

Comment:
"In case of" usually means "if _____ happens". 

"In _the_ case of" (without emphasis) would be more correct here. However, I'd just go with "For icetea:"

Reply:
Agreed,